multinet/db.py

```diff
@@ -282,6 +282,16 @@ def table_fields(workspace: str, table: str, arango: ArangoClient) -> List[str]:
         return []
 
 
+@with_client
+def delete_table(workspace: str, table: str, arango: ArangoClient) -> str:
+    """Delete a table."""
+    space = db(workspace, arango=arango)
+    if space.has_collection(table):
+        space.delete_collection(table)
+
+    return table
```

Comment:
We should check if the table is in a graph here since it's currently possible to break the dependence with the api

Reply:
That's not a bad idea.

My thinking around allowing it in the API is that it's not technically an error to delete a table while a graph depends on it. Imagine, for instance, that you want to replace a bad table with a good one. If we disallow deleting tables that a graph depends on, then the only way to do that is to delete the graph, delete the table, re-upload the table, then re-form the graph. I'd argue that's bad UI.

But, we could definitely have the default be for this delete method to check for a conflict and send back an error code, but also have a `force` parameter that skips the check.

In our use case, however, we still need to check manually for a conflict because of the logic of the delete tables dialog box: that dialog shouldn't allow you to try to delete the tables before it tells you why you can't; it ought to tell you right away. So, I'm for adding a guard to this method, though it doesn't affect our current use case.

Reply:
This sounds like a suitable approach

Reply:
See #228